andela/titan-dev-ah-frontend
# #163518644 User Login

#### What does this PR do?
Add user sign in
#### Description of Task to be completed?
- Convert HTML mockup into react components
- Create Login action creator and reducers
- Connect redux store with Login component
- Add tests for login component, action creators, and reducers
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-user-login-163518644 `
- run `yarn start` and navigate to /login in your browser
-  Submit valid credentials to get a token
- Submit invalid credentials to get errors
#### Any background context you want to provide?
You have to have an email, and your backend server running
#### What are the relevant pivotal tracker stories?
#163518644
